index.html

```diff
@@ -107,93 +137,141 @@ <h3>Multi-Post Stories</h3>
 
             <div class="projects-grid">
 
-                <div class="project-card card-one div-flex">
-                    <h3 class="card-title">Profesional Art Printing Data</h3>
+                <div class="project-card card-one div-flex project-one">
+
+                    <h3 class="card-title">Project One</h3>
+
                     <p class="card-description">A daily selection of privately personalized reads; no accounts or
                         sign-ups required. has been the industry's standard</p>
+                    
+                    <p class="long-description long-description-one">This is the long description for the PROJECT ONE Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita rem quidem maiores. Amet necessitatibus neque architecto quis, eaque corporis nihil. Tenetur magni libero accusamus. Perferendis voluptate ducimus possimus eveniet deserunt.</p>
+
+                    <img class="project-img" src="Projects/Project_1.png" alt="">
+
+                    <a class="see-live-link" href="https://www.freecodecamp.org/certification/fccde5c1dbd-ac5b-4ee7-91c7-386adbe10dcd/responsive-web-design"></a>
+
+                    <a class="see-source-link" href="https://www.freecodecamp.org/certification/fccde5c1dbd-ac5b-4ee7-91c7-386adbe10dcd/responsive-web-design"></a>
+                    
                     <ul id="card-tags" class="card-tags div-flex">
-                        <li class="card-tags-ul-li div-flex">html</li>
-                        <li class="card-tags-ul-li div-flex">Bootstrap</li>
-                        <li class="card-tags-ul-li div-flex">Ruby</li>
+                        <li class="card-tags-ul-li tags-one div-flex">html</li>
+                        <li class="card-tags-ul-li tags-one div-flex">Css</li>
                     </ul>
-                    <a class="btn link-card-button div-flex"
-                        href="https://carreraprogrammer.github.io/carreraprogrammerportfolio/" target="_blank"
-                        rel="noopener">See Project</a>
+
+                    <a class="preview-btn btn link-card-button div-flex project-one">See Project</a>
 
                 </div>
 
```

Comment:
- [ ] You should remove each project card from the HTML page and generate them dynamically with the javascript.